Header header
# Position
geometry_msgs/Vector3 lin
geometry_msgs/Vector3 ang

# Velocity
float32 lin_vel
float32 ang_vel
